### DISCUSSION TITLE
Feature request: Add i18n

### DISCUSSION THREAD

**Discussion by phoenixg** [arthelokyo/astrowind#508] [Ideas]
Please add i18n by leveraging Astro 4's native support for i18n for this template

**Comment by qwert2001qwert-alex** [comment]
And please don't forget about the blog [recipes](https://docs.astro.build/en/recipes/i18n/)

**Comment by alfadormx** [comment]
duplicate of https://github.com/onwidget/astrowind/discussions/642
